src/WebJobs.Script.WebHost/Security/KeyManagement/SecretManager.cs

```diff
@@ -22,7 +22,7 @@ namespace Microsoft.Azure.WebJobs.Script.WebHost
 {
     public class SecretManager : IDisposable, ISecretManager
     {
-        private readonly ConcurrentDictionary<string, Dictionary<string, string>> _secretsMap = new ConcurrentDictionary<string, Dictionary<string, string>>();
+        private readonly ConcurrentDictionary<string, Dictionary<string, string>> _secretsMap = new ConcurrentDictionary<string, Dictionary<string, string>>(StringComparer.OrdinalIgnoreCase);
```

Comment:
In my testing, I noticed that this dictionary was case sensitive. It shouldn't be.

Reply:
👍